server/schedule/operator_controller.go

```diff
@@ -401,9 +421,46 @@ func (s StoreInfluence) ResourceSize(kind core.ResourceKind) int64 {
 	}
 }
 
-// SetOperator is only used for test
+// SetOperator is only used for test.
 func (oc *OperatorController) SetOperator(op *Operator) {
 	oc.Lock()
 	defer oc.Unlock()
 	oc.operators[op.RegionID()] = op
 }
+
+// OperatorWithStatus records the operator and its status.
+type OperatorWithStatus struct {
+	Op     *Operator
+	Status pdpb.OperatorStatus
+}
+
+// OperatorRecords remains the operator and its status for a while.
+type OperatorRecords struct {
+	ttl *cache.TTL
+}
+
+// NewOperatorRecords returns a OperatorRecords.
+func NewOperatorRecords() *OperatorRecords {
+	return &OperatorRecords{
+		ttl: cache.NewTTL(time.Minute, 10*time.Minute),
```

Comment:
Should we make this parameter configurable?

Reply:
I think no need, just keep for a while.